gulpfile.js

```diff
@@ -54,18 +58,36 @@ gulp.task('test-debug', function (done) {
   }, done).start();
 });
 
+gulp.task('sitespeedio', ['serve'], function(done){
+  var run = sitespeedio({
+    urls: [
+      baseUrl + '/app/app.html?file=pf-alert',
+      baseUrl + '/app/app.html?file=pf-tabs',
+      baseUrl + '/app/app.html?file=pf-utilization-bar-chart'
+    ],
+    browser: 'chrome',
+    resultBaseDir: './perf/',
+    connection: 'cable',
+    html: true,
+    budget: {
+      page: {
+        pageWeight:1000000,
+        imageWeight: 300000,
+        jsWeight: 500000,
+        cssWeight: 500000
+      }
+    }
+  });
+  run(done);
+});
+
+gulp.task('perf', function(done){
+  runSequence('build', 'sitespeedio', done)
+});
+
 gulp.task('webpack', ['js'], function() {
-  return gulp.src(['dist/es2015/*/*.js'])
-    .pipe(webpack({
-      resolve: {
-        root: [
-          path.join(__dirname, "dist/es2015/pf-alert"),
```

Comment:
@dlabrecq i've added `root` to the webpack config file so this behavior stays in tact in the `src` and `dist` folder. FWIW I don't think it bad to have relative path resolves in your local repo (just when consuming downstream node_modules).  It would save us from having to keep appending to this configuration, but I'll follow the pattern if we think it useful.